paparazzi/src/main/java/app/cash/paparazzi/internal/ComposeViewAdapter.kt

```diff
@@ -13,4 +14,12 @@ import android.widget.FrameLayout
 internal class ComposeViewAdapter(
   context: Context,
   attrs: AttributeSet
-) : FrameLayout(context, attrs)
+) : FrameLayout(context, attrs) {
+  init {
+    /**
+     * Needed as [android.view.WindowManagerImpl] uses the view root background color to set the WindowManagerImpl view's background color.
+     * If we set this as transparent, the WindowManagerImpl view will be transparent as well and correctly renders the window above content.
+     */
+    setBackgroundColor(Color.TRANSPARENT)
```

Comment:
Change is needed due to a view ordering change in WindowManagerImpl in 14.0.11.

(line 105) https://www.diffchecker.com/fHBDkOTq/
AS diff: https://android.googlesource.com/platform/frameworks/layoutlib/+/f86a9059e4e0fbc6908c32aa88bddc640de86831%5E%21/bridge/src/android/view/WindowManagerImpl.java

Reply:
The timestamp of that change is dated 2021, so I am skeptic of this being the root cause since we have been on later layoutlib versions.  What am I missing?

Reply:
update: I had only clicked on the AS diff per my comment above.  I just viewed the diffcheck link and that makes more sense.  Is it possible you posted the wrong AS diff?

Reply:
yes the one I posted isn't correct but according to the cs.android.com that is the last revision with that line change.

Reply:
Here's the history for WindowManagerImpl: https://cs.android.com/android/platform/superproject/main/+/main:frameworks/layoutlib/bridge/src/android/view/WindowManagerImpl.java;bpv=1

in case there is something we can learn from their changes!